### ISSUE TITLE
Thread safety of factories

### ISSUE THREAD

**Issue by sparty02** [FasterXML/woodstox#120]
Previous versions of woodstox [usage notes](https://github.com/FasterXML/Woodstox4/blob/master/release-notes/USAGE) and [blog posts](http://www.cowtowncoder.com/blog/archives/2006/06/entry_2.html) about it used to advertise that the XML___Factory implementations were thread-safe _after_ the configuration phase, which can provide a big performance boost (and/or avoid having to maintain a pooling implementation) in highly concurrent environments (like web apps).  I don't see thread-safety advertised any longer in any of the docs.

1. Is this a design decision not to guarantee thread-safety of factories any longer?
2. Or is it still a goal, but not a guarantee, which I suppose aligns with what the Stax spec says?

In short, should an implementation be able to rely on thread-safety for woodstox?

**Comment by cowtowncoder** [comment]
Nothing has changed; factories are thread-safe after configuration. Has not changed and should never change.

**Comment by cowtowncoder** [comment]
Would love to get a PR or such; closing this issue itself.
